### DISCUSSION TITLE
ECS Fargate: Add autoscaling without downtime?

### DISCUSSION THREAD

**Discussion by rhoboat** [gruntwork-io/knowledge-base#176] [Help] [answered]
Hey! We were trying move from an ecs fargate service without autoscaling to one with autoscaling. Is there a way to do it with no downtime? Right now, it seems like there's not.

**Comment by denis256** [comment, marked as answer]
Hi,
yes, from my tests I found that when enabling auto-scaling ECS service is recreated which will lead to downtime, as a workaround - I was thinking that can be created separated service with auto scale, traffic shifted to new service and then old one can be removed.
